tests/tests/large_files_tests.robot

```diff
@@ -8,37 +8,28 @@ Resource            src/page_objects/keywords/common.robot
 Resource            src/page_objects/keywords/home_page.robot
 Resource            src/page_objects/keywords/ubuntu.robot
 Resource            src/test_files/test_files.resource
-Resource            /tmp/TestArtifacts/large_test_files.resource
 Library             src/page_objects/libraries/browser_lib.py
 
-Suite Setup         Prepare Test Environment
-Test Setup          Prepare For Test
 Test Teardown       Clean After Test
 
 
-*** Variables ***
-${TEST_FILE}                    ${file_1}
-${EXTRACTION_TIMEOUT}           ${file_1}[extraction_time]
-${PATH_TO_PATTERN_FOLDER}       ${file_1}[pattern_path]
-
-
 *** Test Cases ***
 Extract and validate a large test file
     [Documentation]    Extract and validate a large test file
     [Tags]    daily    regression    large
-    ${downloaded_file_path}    Set Variable    ${DOWNLOAD_PATH}${TEST_FILE}[archive_name].zip
+    ${downloaded_file_path}    Set Variable    ${DOWNLOAD_PATH}${large_file_15}[archive_name].zip
     # Add a file for extraxtion
     Click Add Files Button
-    Upload Test File    ${large_files_upload_path}${TEST_FILE}[name]
+    Upload Test File    ${input_test_files_path}${large_file_15}[name]
```

Comment:
I can be set as: Upload Test File    ${large_file_15}[path]

Reply:
fixed